xla-hlo/benchmark/download_artifacts.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import os
+import pathlib
+import requests
+import sys
+
+from concurrent.futures import ProcessPoolExecutor
+from concurrent.futures import wait
+
+# Add benchmark definitions to the search path.
+sys.path.insert(
+    0,
+    str(
+        pathlib.Path(__file__).parent.parent.parent / "oobi" /
+        "benchmark-definitions" / "python"))
+import data_types, jax_model_definitions, model_dictionary, tf_model_definitions, unique_ids
+
+
+def download_file(source_url: str, save_path: str):
+  pathlib.Path(os.path.dirname(save_path)).mkdir(parents=True, exist_ok=True)
+  r = requests.get(source_url)
+  open(save_path, 'wb').write(r.content)
+  print(f"Downloaded {source_url} to {save_path}")
+
+
+if __name__ == "__main__":
+  argParser = argparse.ArgumentParser()
+  argParser.add_argument("-o",
+                         "--output_dir",
```

Comment:
We can use `type=pathlib.Path` to turn the variable into `pathlib.Path`, which is the recommended class to handle paths

Reply:
Changed